### PR TITLE
Terraform Version Upgrade and Badging Update

### PR DESCRIPTION
**Pull request by JJediny**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@
 coverage/
 crash.log
 node_modules/
+*.terraform.lock.hcl
```
Comment (adborden) [inline .gitignore:9]: Hmm, this should not be ignored. It's the lockfile that makes sure what gets tested is what gets deployed with respect to Terraform and provider versions. As far as I'm aware, this didn't change in 1.0, did it?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,13 @@
-[![18F](https://circleci.com/gh/18F/tts-tech-portfolio.svg?style=svg)](https://app.circleci.com/pipelines/github/18F/tts-tech-portfolio)
+![Terraform Lint and Security Checks](https://github.com/18f/tts-tech-portfolio/actions/workflows/main.yml/badge.svg)
+![Terraform Plan](https://github.com/18f/tts-tech-portfolio/actions/workflows/plan.yml/badge.svg)
+![Terraform Apply](https://github.com/18f/tts-tech-portfolio/actions/workflows/apply.yml/badge.svg)
+![Inspector](https://github.com/18f/tts-tech-portfolio/actions/workflows/inspector.yml/badge.svg)
+
+
+| Active Pull Request Status |
```
Comment (adborden) [inline README.md:7]: Is this supposed to be a table? The formatting looks off on rendering.

**Review by adborden — CHANGES_REQUESTED** [review]
My only concern here is the lockfile being ignored. Was there a reason for that?

File: README.md (hunk 2 above)
Comment (JJediny) [reply README.md]: fixed

File: .gitignore (hunk 1 above)
Comment (JJediny) [reply .gitignore]: fixed

**Review by adborden — APPROVED** [review]

**Comment by adborden** [comment]
🤦  I only looked at your recent commits, but it looks like we lost the actual version 1.0 bump

**Pull request merged**